core/src/test/java/org/frankframework/javascript/JavascriptEngineTest.java

```diff
@@ -12,6 +16,11 @@
 
 class JavascriptEngineTest {
 
+	@BeforeAll
+	public static void setup() {
+		assumeFalse(TestAssertions.isTestRunningOnARM());
```

Comment:
```suggestion
		// Since it's not possible to conditionally check the EnumSource here, just disable all for now...
		assumeFalse(TestAssertions.isTestRunningOnARM(), "uses J2V8 which does not work on ARM");
```